<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="4" total="5" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-09-16T16:00:28 SGT" name="Default suite" finished-at="2025-09-16T16:00:37 SGT" duration-ms="8287">
    <groups>
      <group name="Regression">
        <method signature="MemberTest.advanceSearchMemberFullName(java.util.HashMap)[pri:0, instance:PsslaiMerchantV3.Tests.MemberTest@4b29d1d2]" name="advanceSearchMemberFullName" class="PsslaiMerchantV3.Tests.MemberTest"/>
      </group> <!-- Regression -->
    </groups>
    <test started-at="2025-09-16T16:00:28 SGT" name="Default test" finished-at="2025-09-16T16:00:37 SGT" duration-ms="8287">
      <class name="PsslaiMerchantV3.Tests.MemberTest">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:PsslaiMerchantV3.Tests.MemberTest@4b29d1d2]" started-at="2025-09-16T16:00:29 SGT" name="launchApplication" finished-at="2025-09-16T16:00:33 SGT" duration-ms="4032" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="advanceSearchMemberFullName(java.util.HashMap)[pri:0, instance:PsslaiMerchantV3.Tests.MemberTest@4b29d1d2]" started-at="2025-09-16T16:00:33 SGT" name="advanceSearchMemberFullName" description="Advanced Search Member using Full Name" data-provider="getData" finished-at="2025-09-16T16:00:36 SGT" duration-ms="3735" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{successToast=Filters applied successfully, firstName=Kevin, userPassword=Traxion123!!, fullName=Kevin Edward Gile Pasimio, successMsg=Your request to generate the member list has been received. The file is currently being processed and will be sent to your email shortly., userName=jmendoza+0002@traxiontech.net}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Result doesn't match expected [Kevin Edward Gile Pasimio] but found [No data available in table]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Result doesn't match expected [Kevin Edward Gile Pasimio] but found [No data available in table]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at PsslaiMerchantV3.Tests.MemberTest.advanceSearchMemberFullName(MemberTest.java:73)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- advanceSearchMemberFullName -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:PsslaiMerchantV3.Tests.MemberTest@4b29d1d2]" started-at="2025-09-16T16:00:36 SGT" name="tearDown" finished-at="2025-09-16T16:00:37 SGT" duration-ms="290" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- PsslaiMerchantV3.Tests.MemberTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
